src/Workspaces/Core/Portable/Diagnostics/SkippedHostAnalyzersInfo.cs

```diff
@@ -135,6 +145,28 @@ namespace Microsoft.CodeAnalysis.Diagnostics
                     }
                 }
 
+                if (hostAnalyzer is DiagnosticSuppressor suppressor)
+                {
+                    var suppressionDescriptors = suppressor.SupportedSuppressions;
+                    foreach (var descriptor in suppressionDescriptors)
+                    {
+                        if (projectAnalyzerDiagnosticIds.Contains(descriptor.SuppressedDiagnosticId))
+                        {
+                            // Host analyzer cannot suppress nuget analyzers diagnostics.
+                            skippedDiagnosticIdsBuilder.Add(descriptor.Id);
```

Comment:
@nschuessler This part adds suppression IDs to the skipped list, but you would also need to handle suppressors at the place where these skipped/filtered diagnostic IDs are processed: http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Diagnostics/Extensions.cs,172

Reply:
Actually, I am not sure how would we handle the case where a suppressor reports multiple suppressions, such that some are shared with nuget suppressors and some are unique. We will end up running the suppressor, which will add programmatic suppressors for all its supported suppressions - I don't know of a good way to remove host suppressor suppressions for the shared suppression IDs with nuget suppressors.

I think lets make this more simple - if `projectSuppressedDiagnosticIds.Contains(descriptor.SuppressedDiagnosticId)` is true below, lets just return `false` so we only execute those host suppressors that have all unique suppressions.

Reply:
@mavasani  I have solved this duplicate suppressions problem by inventorying the suppressed id's on the nuget suppressors. If a nuget supresses the same id as a host suppressor, the nuget will get priority, and the host suppressor suppressions will be excluded.

Reply:
@nschuessler The part where you collect `projectSuppressedDiagnosticIds` is fine. I am suggesting simplifying the part that uses it as per https://github.com/dotnet/roslyn/pull/47486#discussion_r485059119

Reply:
Per discussion, due to other changes, changes to Extension.cs no longer / not necessary because suppressors won't complete anymore.

Reply:
Should be fixed.